### ISSUE TITLE
Install Scripts: WordPress - Some small improvements to the install form

### ISSUE THREAD

**Issue by shoulders** [virtualmin/virtualmin-gpl#732]
|  SYSTEM INFORMATION||
|----------------------|---------------------------|
|  OS type and version |     Ubuntu Linux 22.04.3     |
|  Webmin version      |     2.105      |
|  Usermin version     |     2.005     |
|  Virtualmin version  |     7.9.0       |
|  Theme version       |     21.09.5      |
|  Package updates     |     All installed packages are up to date   |

### background

I wanted to install wordpress so i thought I would use the script

### The issues

![image](https://github.com/virtualmin/virtualmin-gpl/assets/319997/8420b1bd-8797-4d3c-bb8d-66522b18c4d7)

-  `Use WordPress CLI tool for complete setup`
   -  I dont know what this means. If it is on will Virtualmin fully setup wordpress using the CLI? If it is off what will not be setup.
   - Can you add a tooltip clarify this option as I bet most people just leave it on.
- on the password field can you add a password generator button
![image](https://github.com/virtualmin/virtualmin-gpl/assets/319997/1e6ce553-f169-4d88-86da-c01d10e65163)

Thanks

**Comment by jcameron** [comment]
You make a good point - maybe we don't need that option if the CLI tool is available, because it's always the superior choice.

@iliajie is it the theme that adds the password generator to those fields?

**Comment by iliajie** [comment]
> we don't need that option if the CLI tool is available, because it's always the superior choice.

Yes, I just dropped the old code and made it always use WP-CLI in this patch https://github.com/virtualmin/virtualmin-gpl/commit/f3e511fd69b430c9fd6c0488522b58bf68d193b1.



> is it the theme that adds the password generator to those fields?

Yes, theme adds the generate password buttons. Although the script installer can define it too as I added support for it some time ago https://github.com/virtualmin/virtualmin-gpl/commit/718282db5a0944cc98da7474d85ca1e65c90c8c1.

**Comment by shoulders** [comment]
> Fix to pre-generate secure password if domain password isn't good enough

The domain password should not be used. It is bad security practice to use the same password multiple times, especially on the same system.

**Comment by iliajie** [comment]
> The domain password should not be used. It is bad security practice to use the same password multiple times, especially on the same system.

A domain password won't be used, if Virtualmin is configured not to use plain-text passwords.

**Comment by iliaross** [comment]
Reopened, as this was closed by an automated cleanup run. @shoulders, sorry for the noise, and thanks for the contribution. We definitely didn't mean any disrespect.

If it turns out the issue is already resolved for you, you're welcome to close it.
